tests/unit/src/flixel/ui/FlxButtonTest.hx

```diff
@@ -14,6 +17,7 @@ class FlxButtonTest extends FlxTest
 	{
 		button = new FlxButton();
 		destroyable = button;
+		FlxG.mouse.playback(new MouseRecord(0, 0, FlxInputState.RELEASED, 0)); // put the mouse in the upper-left corner to prepare for highlight testing
```

Comment:
Hm, yeah.. I thought resetGame would completely recreate all instances in FlxG, guess it's actually far from that. All this global stuff is pretty annoying for unit testing... :/

We should at least handle the mouse reset in `FlxTest#resetGame()`, so it's not just a random hack, but happens consistently. Perhaps we can even manually call `FlxG.init()`, so things _really_ get reset (new `FlxMouse` instance).